temporian/core/data/duration.py

```diff
@@ -35,23 +35,23 @@ def milliseconds(value: float) -> Duration:
 
 
 def seconds(value: float) -> Duration:

```

Comment:
User will also want to feed values themselves. e.g.:

b = tp.moving_count(a, 5)


I think it would be better for Duration to be Union[float,int], and for attributes to do the conversion if necessary (as discussed during meeting).